ciao-controller/main.go

```diff
@@ -61,6 +68,7 @@ var servicePassword = ""
 var volumeAPIPort = block.APIPort
 var computeAPIPort = compute.APIPort
 var imageAPIPort = osimage.APIPort
+var controllerAPIPort = api.Port
```

Comment:
not so far. I've tested with single VM - it does require some modification to our fake identity server, but not for this reason. The modification is required because of the cli changes I made to support privileged vs. non-privileged routes. Our fake identity service sets the token to always be in the project "ilovepuppies", but I made privileged actions require the "admin" project.